Presentations/03-data-visualization.Rmd

```diff
@@ -129,22 +143,15 @@ Similarly to previous sessions, you can find some references at the end of this
 # Packages
 library(tidyverse)
 library(plotly)
+library(here)
 
 # Load CSV data
-# Replace with where your data is
-
-FolderPath <- file.path("YOUR/FOLDER/PATH")
-
-dataWorkFolder    <- file.path(projectFolder,"DataWork")
-Data              <- file.path(dataWorkFolder,"DataSets")
-finalData         <- file.path(Data,"Final")
-
-whr_panel <- readRDS(file.path(finalData, "whr_panel.RDS"))
+here::here()
```

Comment:
this line might be missing the command to load the csv file

Reply:
This was fixed in #63